### PR TITLE
Create zh_cn.json

### PR DESCRIPTION
**Pull request by 200542053**

### CODE REVIEW

File: src/main/resources/assets/fabriblocks/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	"block.fabriblocks.mossy_brick_wall": "苔砖墙",
+	"block.fabriblocks.snow_brick_wall": "雪砖墙",
+	"block.minecraft.lectern": "橡木讲台",
+	"block.fabriblocks.spruce_lectern": "云杉木讲台",
+	"block.fabriblocks.birch_lectern": "白桦木讲台",
+	"block.fabriblocks.jungle_lectern": "丛林木讲台",
+	"block.fabriblocks.acacia_lectern": "金合欢木讲台",
+	"block.fabriblocks.dark_oak_lectern": "深色橡木讲台",
+	"block.fabriblocks.granite_pressure_plate": "花岗岩压力板",
+	"block.fabriblocks.diorite_pressure_plate": "花岗岩压力板",
```
Comment (YanisBft) [inline src/main/resources/assets/fabriblocks/lang/zh_cn.json]: It's the same translation as for granite pressure plate ^^

File: src/main/resources/assets/fabriblocks/lang/zh_cn.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	"block.fabriblocks.andesite_pressure_plate": "安山岩压力板",
+	"block.fabriblocks.granite_button": "花岗岩按钮",
+	"block.fabriblocks.diorite_button": "花岗岩按钮",
```
Comment (YanisBft) [inline src/main/resources/assets/fabriblocks/lang/zh_cn.json]: It's the same translation as for granite button ^^

**Review by YanisBft — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/fabriblocks/lang/zh_cn.json (hunk 1 above)
Comment (200542053) [reply src/main/resources/assets/fabriblocks/lang/zh_cn.json]: I have fixed these two bugs :)

**Comment by YanisBft** [comment]
Thanks a lot for the translation! I'll merge it when I get to work on the next update :)

**Review by YanisBft — APPROVED** [review]

**Pull request merged**